pkg/rpm/salt.spec

```diff
@@ -482,8 +486,12 @@ if [ $1 -lt 2 ]; then
   # ensure hmac are up to date, master or minion, rest install one or the other
   # key used is from openssl/crypto/fips/fips_standalone_hmac.c openssl 1.1.1k
   if [ $(cat /etc/os-release | grep VERSION_ID | cut -d '=' -f 2 | sed  's/\"//g' | cut -d '.' -f 1) = "8" ]; then
```

Comment:
you need this line, since only need it on RedHat 8, e.g. RHEL 9 doesn't use 1.1.1

Reply:
I'm not taking it out

Reply:
d'oh